packages/@okta/vuepress-site/guides/custom-error-pages/customization-examples/index.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Customization Examples
+---
+Use the following customization examples to help you personalize your error page by extending the look and feel of the page with your own stylesheet (CSS) and defining per-application error messages.
```

Comment:
We dont support per-application error messages

Reply:
Good catch - forgot to remove reference to it after our discussion.

Reply:
Nice catch - thank you. Didn't remove all of the references to application specific content after we talked.